plugins/modules/zos_fetch.py

```diff
@@ -91,26 +92,26 @@
     suboptions:
       from:
         description:
-            - The encoding to be converted from
+            - The encoding to be converted from.
         required: true
         type: str
       to:
         description:
-            - The encoding to be converted to
+            - The encoding to be converted to.
         required: true
         type: str
 notes:
     - When fetching PDS(E) and VSAM data sets, temporary storage will be used
       on the remote z/OS system. After the PDS(E) or VSAM data set is
-      successfully transferred, the temporary data set will deleted. The size
+      successfully transferred, the temporary data set will be deleted. The size
       of the temporary storage will correspond to the size of PDS(E) or VSAM
       data set being fetched. If module execution fails, the temporary
       storage will be cleaned.
     - To prevent redundancy, additional checksum validation will not be done
       when fetching PDS(E) because data integrity checks are done through the
       transfer methods used. As a result, the module response will not include
       C(checksum) parameter.
-    - All data sets are always assumed to be in catalog. If an uncataloged data
+    - All data sets are always assumed to be in the catalog. If an uncataloged data
```

Comment:
I am good with either if you see this in the future:
All data sets are always assumed to be in the catalog.
All data sets are always assumed to be cataloged.